setup.py

```diff
@@ -6,16 +6,16 @@
 
 setup(
     name='imagestore',
-    version='3.1.0',
+    version='3.2.0',
     packages=find_packages(),
     install_requires=[
-        'django',
-        'pillow',
-        'sorl-thumbnail',
+        'django>=1.11',
+        'pillow>=5.4.1',
```

Comment:
```suggestion
        'pillow',
```
I think we can use any version of pillow here.